modules/installation-configuration-parameters.adoc

```diff
@@ -312,6 +326,16 @@ ifdef::osp[]
    }
 }
 ----
+
+|`platform.openstack.clusterOSImage`
+|The location from which the installer downloads the {op-system} image.
+
+You may need to set this parameter to perform an installation in a restricted network.
```

Comment:
It saus "you may need it" but this is required. I'd suggest something like the following:

If not specified, the installer will download the image from a predetermined location described in https://github.com/openshift/installer/blob/master/data/data/rhcos.json 

This is required to perform an installation in a restricted networking environment.

@Fedosin FYI

Reply:
Is there a reason to specify where the installer fetches the image from by default? If not, I'd prefer to leave it out--space is at a premium in the param tables. 

Will strike "may."